Makefile

```diff
@@ -170,10 +176,18 @@ undeploy-with-olm: ## Undeploy controller from the K8s cluster
 
 # Make target to ignore (git checkout) changes if there are only timestamp changes in the bundle
 checkout-bundle-timestamp:
-	(git diff --quiet --ignore-matching-lines createdAt bundle && git checkout --quiet bundle) || true
+	(git diff --quiet --ignore-matching-lines createdAt bundle/odf-operator && git checkout --quiet bundle/odf-operator) || true
+	(git diff --quiet --ignore-matching-lines createdAt bundle/odf-dependencies && git checkout --quiet bundle/odf-dependencies) || true
 
 .PHONY: bundle
 bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metadata, then validate generated files.
+	# Dependencies bundle
+	$(KUSTOMIZE) build config/bundle | $(OPERATOR_SDK) generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS) \
+		--output-dir bundle/odf-dependencies --package odf-dependencies
+	$(OPERATOR_SDK) bundle validate bundle/odf-dependencies
+	@mv bundle.Dockerfile bundle.deps.Dockerfile
```

Comment:
why is this `mv` needed here?

Reply:
mv is needed to have 2 different docker files for different bundles.

Reply:
still don't get it, bundle.Dockerfile contains odf-op content and moving it to bundle.deps.Dockerfile doesn't seem correct, both these files are part of this PR?

Reply:
bundle.Dockerfile is created via operator-sdk while running `generate bundle` and we are generating 2 bundles. First one is for deps and second one is for the odf-operator. I am doing a mv on the first bundle.Dockerfile. 

PS: operator-sdk always create the file named bundle.Dockerfile.